app/dossierPrograms/dossierPrograms.services.js

```diff
@@ -33,6 +20,13 @@ dossierProgramsModule.factory("dossiersProgramsFactory", [
     },
 ]);
 
+var qryProgramStageSections =
+    dhisUrl +
+    "programStages/:programStageId?fields=id,displayName,repeatable,sortOrder," +
+    "programStageSections[id,displayName,dataElements[id,displayName,displayFormName,displayDescription,valueType," +
+    "optionSetValue,optionSet[name,options[displayName]]]],programStageDataElements[dataElement[id,displayName," +
+    "displayFormName,displayDescription,valueType,optionSetValue,optionSet[name,options[displayName]]]]&paging=false";
+
```

Comment:
Same as the other PR, it's hard to follow which fields are being requested.

Reply:
Fixed following the previous PR proposal 1 style.